acceptance/tests/peering/peering_connect_test.go

```diff
@@ -53,10 +53,14 @@ func TestPeering_Connect(t *testing.T) {
 
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
+			fmt.Println("starting test")
```

Comment:
replace with t.Log?

Reply:
whoops thanks i forgot to remove those, I was just using those locally so i could see what was going on while running the test!